### PR TITLE
Make jQuery Optional

### PR DESCRIPTION
**Pull request by sandstrom**
[Link to rendered RFC](https://github.com/sandstrom/rfcs/blob/master/text/0000-jquery-optional.md)

### CODE REVIEW

File: text/0000-jquery-optional.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+# Alternatives
+
+Wait 1-2 years, then axe jQuery.
+
+# Unresolved questions
+
+- How would the core team like to handle the 'make it optional' switch? May affect the build process,
+library requirements etc.
+- The FF11 requirement, what is the minimum supported version of FF? Is it fine to drop? (FF is on ~38 now)
```
Comment (dukex) [inline text/0000-jquery-optional.md]: FF11

**Comment by mike1o1** [comment]
One approach the Angular team takes is to use Google's [jqLite](https://code.google.com/p/jqlite/) library if jQuery itself isn't loaded. Is it worth investigating to see if a similar approach can be taken for Ember? 

This can also be exposed for developers. If a developer invokes `Ember.$` or `this.$()` within a view and jQuery is not available, Ember would return an instance of jqLite. Angular's equivalent to `Ember.$` is `angular.element`, which is documented [here](https://docs.angularjs.org/api/ng/function/angular.element), in case you wanted some prior art to look at.

**Comment by locks** [comment]
Related: https://github.com/emberjs/ember.js/issues/3791

**Comment by runspired** [comment]
Related https://github.com/emberjs/rfcs/pull/86 removes most of the reliance on jQuery for eventing.

In my current implementation, `jQuery.event.fix(e)` is used to normalize the event. This is something that could be pulled out as a micro-lib exposing the needed jquery AMD module.

https://github.com/runspired/ember-capture-eventing/blob/master/addon/event_dispatcher.js#L379

**Comment by ming-codes** [comment]
Related: https://github.com/rwjblue/ember-native-dom-event-dispatcher

Haven't tried it yet, but assuming it works, it's a bridge to be rid of jQuery.

**Comment by sandstrom** [comment]
Closing this RFC, since work is [now](https://github.com/emberjs/ember.js/issues/16058) [underway](https://github.com/emberjs/data/issues/5320) to make jQuery optional.

**Pull request closed without merging**